benchmarker/nvprof.py

```diff
@@ -2,22 +2,22 @@
 import subprocess
 from benchmarker.util import abstractprocess
 
-exp1_re = re.compile('(\d+)\s+flop\_count\_sp\s+Floating\s+Point\s+Operations\(Single Precision\)\s+(?:(\d+)\s+){3}')
-exp2_re = re.compile('(\d+)\s+flop\_count\_sp\s+Floating\s+Point\s+Operations\(Single Precision\)\s+(?:(\d+\.\d+e[\+|\-]\d+)\s+){3}')
+nvprof_expr1_re = re.compile('(\d+)\s+flop\_count\_sp\s+Floating\s+Point\s+Operations\(Single Precision\)\s+(?:(\d+)\s+){3}')
```

Comment:
What does this (and the next) `re` do? What expression/value do they match? Flops? Put that in the name of the regexp!

Reply:
The first regex captures operations in whole number and the second one captures operations written in scientific notation. Ideally, I wanted to use one regex that captures both but it was not working so kept as it for the time being but will update once I figure that out.
Ok, making regexp names highlighting what I mentioned above.

Reply:
yey! :partying_face: